src/Microsoft.ML.Vision/ImageClassificationTrainer.cs

```diff
@@ -1322,9 +1323,9 @@ private void VariableSummaries(ResourceVariable var)
         private static TensorFlowSessionWrapper LoadTensorFlowSessionFromMetaGraph(IHostEnvironment env, Architecture arch)
         {
             var modelFileName = ModelFileName[arch];
-            var modelFilePath = Path.Combine(_resourcePath, modelFileName);
+            var modelFilePath = Path.Combine((env as IHostEnvironmentInternal).TempFilePath, "MLNET", modelFileName);
```

Comment:
It maybe better to make this an instance method and continue to use `_resourcePath`.

Reply:
I tried that and it caused other issues. Let me make that change again and I'll give more details. IIRC though it has to do with us parsing the graph before we have instantiated it here.

Reply:
Nevermind, I was thinking of something else. Changed it to an instance method and reverted to `_resourcePath`.